Panic on Windows during type checking: not yet implemented

## Setup

On my project, CI fails for Windows while type-checking the tests. I do not have a windows machine right now where I could reproduce this locally.

## Reproduction

It only happens approximately 1 in 10 times. It seems flaky—running the job again on the same commit usually fixes it.

This is the script that CI runs on Windows.

```sh
git clone git@github.com:grammyjs/grammY.git
cd grammY
deno task check
deno task test
```

The actual CI definition is here: https://github.com/grammyjs/grammY/blob/ec08af54a1cf441cda2034353f7ecc2a9d1b8981/.github/workflows/deno.yml#L46

## Crash Logs

I have added `RUST_BACKTRACE=full` to CI and now I see this:

```sh
============================================================
Deno has panicked. This is a bug in Deno. Please report this
at https://github.com/denoland/deno/issues/new.
If you can reliably reproduce this panic, include the
reproduction steps and re-run with the RUST_BACKTRACE=1 env
var set and include the backtrace in your report.

Platform: windows x86_64
Version: 2.0.5
Args: ["C:\\hostedtoolcache\\windows\\deno\\2.0.5\\x64\\deno.exe", "test", "--seed=123456", "--parallel", "--allow-import", "./test/"]

thread 'tokio-runtime-worker' panicked at runtime\errors.rs:626:35:
not yet implemented
stack backtrace:
   0:     0x7ff6cf00e405 - onig_get_string_end_by_callout_args
   1:     0x7ff6cd7048a9 - onig_get_start_by_callout_args
   2:     0x7ff6cefe9ee7 - onig_get_string_end_by_callout_args
   3:     0x7ff6cf011cc1 - onig_get_string_end_by_callout_args
   4:     0x7ff6cf01182d - onig_get_string_end_by_callout_args
   5:     0x7ff6cd57d238 - onig_get_capture_tree
   6:     0x7ff6cf0127eb - onig_get_string_end_by_callout_args
   7:     0x7ff6cf01265f - onig_get_string_end_by_callout_args
   8:     0x7ff6cf0125ef - onig_get_string_end_by_callout_args
   9:     0x7ff6cf0125d6 - onig_get_string_end_by_callout_args
  10:     0x7ff6d0afb8f4 - onig_unicode_define_user_property
  11:     0x7ff6d0afb9ad - onig_unicode_define_user_property
  12:     0x7ff6ce1a647a - node_api_get_module_file_name
  13:     0x7ff6cd260983 - onig_get_capture_tree
  14:     0x7ff6cd89db78 - onig_get_start_by_callout_args
  15:     0x7ff6cea11c8e - onig_builtin_fail
  16:     0x7ff6d077dd06 - CrashForExceptionInNonABICompliantCodeRange
  17:     0x7ff6d077bede - CrashForExceptionInNonABICompliantCodeRange
  18:     0x7ff6d077bede - CrashForExceptionInNonABICompliantCodeRange
  19:     0x7ff6d077bede - CrashForExceptionInNonABICompliantCodeRange
  20:     0x7ff6d077bede - CrashForExceptionInNonABICompliantCodeRange
  21:     0x7ff6d077bede - CrashForExceptionInNonABICompliantCodeRange
  22:     0x7ff6d077bede - CrashForExceptionInNonABICompliantCodeRange
  23:     0x7ff6d07bfecd - CrashForExceptionInNonABICompliantCodeRange
  24:     0x7ff6d0779a5c - CrashForExceptionInNonABICompliantCodeRange
  25:     0x7ff6d07795af - CrashForExceptionInNonABICompliantCodeRange
  26:     0x7ff6cf547df8 - onig_get_string_end_by_callout_args
  27:     0x7ff6cf5489d5 - onig_get_string_end_by_callout_args
  28:     0x7ff6cf548b33 - onig_get_string_end_by_callout_args
  29:     0x7ff6cf550151 - onig_get_string_end_by_callout_args
  30:     0x7ff6cf54f8aa - onig_get_string_end_by_callout_args
  31:     0x7ff6cf54f6be - onig_get_string_end_by_callout_args
  32:     0x7ff6cf54f6be - onig_get_string_end_by_callout_args
  33:     0x7ff6cf54f6be - onig_get_string_end_by_callout_args
  34:     0x7ff6cf54f6be - onig_get_string_end_by_callout_args
  35:     0x7ff6cf54f6be - onig_get_string_end_by_callout_args
  36:     0x7ff6cf54f6be - onig_get_string_end_by_callout_args
  37:     0x7ff6cf54f6be - onig_get_string_end_by_callout_args
  38:     0x7ff6cf54f31f - onig_get_string_end_by_callout_args
  39:     0x7ff6cf54955c - onig_get_string_end_by_callout_args
  40:     0x7ff6cf3d11dd - onig_get_string_end_by_callout_args
  41:     0x7ff6cf3bd2c0 - onig_get_string_end_by_callout_args
  42:     0x7ff6cd9899cd - onig_get_start_by_callout_args
  43:     0x7ff6cce2b877 - uv_mutex_destroy
  44:     0x7ff6cd550b44 - onig_get_capture_tree
  45:     0x7ff6ccf23c09 - onig_get_regex_by_callout_args
  46:     0x7ff6cd62b4c5 - onig_get_capture_tree
  47:     0x7ff6cd090f10 - onig_get_regex_by_callout_args
  48:     0x7ff6cd0ba230 - onig_get_regex_by_callout_args
  49:     0x7ff6ccf1fccf - onig_get_regex_by_callout_args
  50:     0x7ff6cd62b4c5 - onig_get_capture_tree
  51:     0x7ff6cd094e9c - onig_get_regex_by_callout_args
  52:     0x7ff6cf191bb9 - onig_get_string_end_by_callout_args
  53:     0x7ff6cf19175d - onig_get_string_end_by_callout_args
  54:     0x7ff6cf00a1fc - onig_get_string_end_by_callout_args
  55:     0x7ffa09964cb0 - BaseThreadInitThunk
  56:     0x7ffa0a85ecdb - RtlUserThreadStart
Error: Process completed with exit code 1.
```

You can view these logs in GitHub Actions as long as they are stored: https://github.com/grammyjs/grammY/actions/runs/11757188343/job/32754154943#step:5:518

--- a/runtime/errors.rs
+++ b/runtime/errors.rs
@@ -623,7 +623,7 @@ fn get_ffi_call_error_class(e: &CallError) -> &'static str {
 fn get_webstorage_class_name(e: &WebStorageError) -> &'static str {
   match e {
     WebStorageError::ContextNotSupported => "DOMExceptionNotSupportedError",
-    WebStorageError::Sqlite(_) => todo!(),
+    WebStorageError::Sqlite(_) => "Error",
     WebStorageError::Io(e) => get_io_error_class(e),
     WebStorageError::StorageExceeded => "DOMExceptionQuotaExceededError",
   }

